README.md

```diff
@@ -309,9 +309,10 @@ of the actual exit code.
 
 ### Windows Compatibility
 
-Due to the blocking nature of `STDIN`/`STDOUT`/`STDERR` pipes on Windows we we can 
-not guarantee this package works as expected on Windows directly. However this package 
-does work on [`Windows Subsystem for Linux`](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux) 
+Due to the blocking nature of `STDIN`/`STDOUT`/`STDERR` pipes on Windows we can 
+not guarantee this package works as expected on Windows directly. As such when 
+instantiating `Process` it throws an exception when on Windows directly. 
```

Comment:
The doubling of "directly" at the end of the 2 sentences could probably be avoided.